src/assignments/html-me-something.rst

```diff
@@ -0,0 +1,93 @@
+.. _html-me-something:
+
+Assignment #4: HTML Me Something
+=================================
+
+You’ve learned a bit of HTML and some CSS, but you have likely only used
+it in bits and pieces so far, adding or modifying content in exercises
+or pre-existing files. Here, you are going to take another step forward
+by building an entire page from scratch. You will also get some practice
+using Git.
+
+There are two parts to this exercise, one focused on HTML and another
+focused on CSS. HTML makes up the *structure and content* of web pages,
+while CSS dictates the *visual style*.
+
+Best practices dictate that *content* and *style* should be kept as separate as
+possible. To that end, we will build the HTML portion of our page first,
+and afterwards we will add a few styles with CSS. We do this to avoid using
+HTML tags to change the general appearance of our page. For example, what if we
+want all of our main headings to be *red*? We can either add this style one
+time in the CSS file, or we must include ``style="color:red"`` in EVERY h1 tag.
+Especially for large websites, CSS provides the best place to control the
+overall appearance of a page.
+
+Sections:
+----------
+
+#. :ref:`Getting Started <getting-started>`
+#. :ref:`Part 1: HTML <html-me-part1>`
+#. :ref:`Part 2: CSS <html-me-part2>`
+#. :ref:`Submitting Your Work <submitting-your-work>`
+
+.. _getting-started:
+
+Getting started
+----------------
+
+For this project, we suggest working on it one section at a time. 
+
+You will be pushing your final assignment up to GitHub; creating a repository for your work should be your first step.
```

Comment:
So I followed the instructions and they are great! I just had a couple of moments of confusion:
1. I chose a console application out of all of the options, because I have seen what happens when I choose web application, but some students might choose a web app because it is HTML and CSS. Maybe ensuring they know to choose a console app in this instance could be helpful?
2. I think at some point the instructions refer to them starting out with an empty directory. Should they delete ``Program.cs`` or keep it?

Reply:
yes, in the html chapter we talk about how to make this type of project.  but i did add a short re-cap as a reminder.